scaletest/terraform/action/prometheus_helm_values.tftpl

```diff
@@ -0,0 +1,44 @@
+alertmanager:
+  enabled: false
+blackboxExporter:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: "cloud.google.com/gke-nodepool"
+            operator: "In"
+            values: ["${nodepool}"]
+operator:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: "cloud.google.com/gke-nodepool"
+            operator: "In"
+            values: ["${nodepool}"]
+prometheus:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: "cloud.google.com/gke-nodepool"
+            operator: "In"
+            values: ["${nodepool}"]
+  externalLabels:
+    cluster: "${cluster}"
+  persistence:
+    enabled: true
+    storageClass: standard
```

Comment:
Do we need cluster-local persistence for Prometheus if we're going to be remote-writing the data anyway?